pyquil/tests/test_paulis.py

```diff
@@ -726,18 +723,14 @@ def test_simplify():
 def test_dont_simplify():
     t1 = sZ(0) * sZ(1)
     t2 = sZ(2) * sZ(3)
-    with pytest.warns(UnequalLengthWarning):
-        assert (t1 + t2) != 2 * sZ(0) * sZ(1)
+    assert (t1 + t2) != 2 * sZ(0) * sZ(1)
 
 
 def test_simplify_warning():
     t1 = sZ(0) * sZ(1)
     t2 = sZ(1) * sZ(0)
-    with pytest.warns(UserWarning) as e:
-        tsum = t1 + t2
-
+    tsum = t1 + t2
     assert tsum == 2 * sZ(0) * sZ(1)
-    assert str(e[0].message).startswith("The term Z1Z0 will be combined with Z0Z1")
```

Comment:
xD

Reply:
Yeah, the warning seemed unnecessary anyway, but message-based checks are less than ideal. :)

I gather you weren't sad to see it go?

Reply:
I won't lose any sleep over it haha